Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/impl/ConceptManager.java

```diff
@@ -572,4 +574,10 @@ private int getConceptsCountForConceptList(String listName) {
         alternativeIdService.addAlternativeIds(conceptEntries);
         return conceptEntries;
     }
+
+    @Override
+    public void updateWordnetConceptInIndex(ConceptEntry entry, String userName)
```

Comment:
just call it ```updateIndex``` since this would work for any entry. If the concept already exist, will the old index entry be updated? Or would there be a second entry?